codec/console_reader_test.go

```diff
@@ -28,7 +28,7 @@ import (
 
 	"github.com/golang/protobuf/proto"
 	"github.com/streamingfast/firehose-aptos/types"
-	pbaptos "github.com/streamingfast/firehose-aptos/types/pb/aptos/extractor/v1"
+	pbaptos "github.com/streamingfast/firehose-aptos/types/pb/aptos/reader/v1"
```

Comment:
Aptos is particular as they manage the `.proto` file themselves, no changes should be made to anything under `github.com/streamingfast/firehose-aptos/types/pb`.